scripts/deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Copyright 2017 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export REGISTRY=quay.io/kubernetes-service-catalog
+
+if [[ -n "${TRAVIS_TAG:-}" ]]; then
```

Comment:
disclosure: I copied and altered this code from [here](https://github.com/kubernetes/helm/blob/master/scripts/ci/deploy.sh#L21-L28) and kept it because it works for them in Circle CI.

However, it looks redundant. Will remove